packages/eden/src/resolve.ts

```diff
@@ -357,15 +362,15 @@ export async function resolveEdenRequest<
       }
 
       if (field instanceof FileList) {
-        for (let i = 0; i < field.length; i++)
-          formData.append(key as any, await createNewFile((field as any)[i]))
+        for (let index = 0; index < field.length; index++)
+          formData.append(key as any, await createNewFile((field as any)[index]))
 
         continue
       }
 
       if (Array.isArray(field)) {
-        for (let i = 0; i < field.length; i++) {
-          const value = (field as any)[i]
+        for (let index = 0; index < field.length; index++) {
```

Comment:
I don't really like verbalizing `index` in these kinds of for loops because it's just a really universal pattern to do `for (let i = 0; ...)` and `index` doesn't really add any additional clarity.